aws/resource_aws_rds_cluster_instance.go

```diff
@@ -292,6 +311,7 @@ func resourceAwsRDSClusterInstanceRead(d *schema.ResourceData, meta interface{})
 
 	d.Set("publicly_accessible", db.PubliclyAccessible)
 	d.Set("cluster_identifier", db.DBClusterIdentifier)
+	d.Set("engine", db.Engine)
```

Comment:
We should also read the engine_version here

Reply:
will fix it too